e2e/test/iothub/FileUploadE2ETests.cs

```diff
@@ -25,6 +26,7 @@ public class FileUploadE2ETests : E2EMsTestBase
 
         [LoggedTestMethod]
         [TestCategory("LongRunning")]
+        [Obsolete]
```

Comment:
I'd rather suppress with a reason for the benefit of future code readers.

Reply:
The CS suppression will need to be inline with #pragmas. I opted for this as it keeps us from having to push/pop these warning checks on the preprocessor stack.

That being said, I'll add some comments around the methods.